### PR TITLE
[11.0][MIG] analytic_partner: Migration to 11

### PR DESCRIPTION
**Pull request by cristinamartinrod**
This module adds a commercial partner on each analytic item for allowing to have another dimension for data analysis. It also handles the proper propagation of this field to the created analytic
entries when validating invoices.

cc @tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

File: analytic_partner/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* `Tecnativa <https://www.tecnativa.com>`__:
+
+  * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```
Comment (pedrobaeza) [inline analytic_partner/readme/CONTRIBUTORS.rst]: Old email

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I have squashed a bit the commit history, generate the README and remove the old email.

File: analytic_partner/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+To configure this module, you need to:
+
+#. Go to **Setting > Users > Users** and check you have as at least "Adviser"
+in your user profile and have checked the "Analytic Accounting" access right.
```
Comment (cubells) [inline analytic_partner/readme/CONFIGURE.rst]: You must indent with 3 spaces to format correctly this file.

File: analytic_partner/readme/CONFIGURE.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
+To configure this module, you need to:
+
+#. Go to **Setting > Users > Users** and check you have as at least "Adviser"
```
Comment (cubells) [inline analytic_partner/readme/CONFIGURE.rst:3]: you have checked "Show Full Accounting Features"

**Review by cubells — APPROVED** [review]
Minor changes

:+1: 

LGTM

code review and test

File: analytic_partner/readme/CONFIGURE.rst (hunk 3 above)
Comment (cristinamartinrod) [reply analytic_partner/readme/CONFIGURE.rst:3]: Thanks. Done!

**Pull request merged**